cluster-scope/base/apps/daemonsets/kvm-device-plugin/machineconfig.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: machineconfiguration.openshift.io/v1
```

Comment:
Please move these resources to a new folder in [`cluster-scope/base/machineconfiguration.openshift.io/machineconfigs`](https://github.com/operate-first/apps/tree/master/cluster-scope/base/machineconfiguration.openshift.io/machineconfigs). This new folder should be named `80-enable-nested-virt`. Resource of `machineconfiguration.openshift.io` api and `MachineConfig` kind should not be placed in `apps` api folder under `daemonsets` kind.

Reply:
Thanks @tumido , I'm new to this. I'll read up on the suggested docs.